client/src/components/Rating.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import Rating from 'react-rating';
+import EmptyAxt from '../assets/RatingAxtEmpty.png';
+import FullAxt from '../assets/RatingAxtFull.png';
+import { saveToLocalStorage, loadFromLocalStorage } from '../lib/localStorage';
+
+export default function RatingStar() {
+  const [rating1, setRating1] = useState(loadFromLocalStorage('Rating') ?? []);
+
+  useEffect(() => {
+    saveToLocalStorage('Rating', rating1);
+  }, [rating1]);
+
+  function getDivide(rating1) {
+    const sum = rating1.reduce((a, b) => a + b, 0);
+    const divide = sum / rating1.length;
+    return divide.toFixed(1);
+  }
+
+  function handleRatingonClick(rate) {
```

Comment:
You could change the function name to handleRating**O**nClick to stay consistent with the rest.